src/actions/breakpoints/syncBreakpoint.js

```diff
@@ -59,16 +59,16 @@ export async function syncClientBreakpoint(
   const existingClient = client.getBreakpointByLocation(generatedLocation);
 
   if (existingClient) {
-    const newGeneratedLocation = existingClient.actualLocation;
-    const newLocation = await sourceMaps.getOriginalLocation(
-      newGeneratedLocation
-    );
+    // const newGeneratedLocation = existingClient.actualLocation;
```

Comment:
im not sure this will work as intended, the server will still have a breakpoint that moved, but now we will not represent it on the client